ui/components/component-library/avatar-token/avatar-token.stories.js

```diff
@@ -93,18 +116,18 @@ export const ColorBackgroundColorAndBorderColor = (args) => (
       {...args}
       backgroundColor={COLORS.GOERLI}
       borderColor={COLORS.GOERLI}
-      tokenName="G"
+      name="G"
       color={COLORS.PRIMARY_INVERSE} // TODO: This will have to be added to the AvatarBase component as a prop so we can change the color of the text and to the base avatar
     />
     <AvatarToken
       {...args}
       backgroundColor={COLORS.SEPOLIA}
       borderColor={COLORS.SEPOLIA}
-      tokenName="G"
+      name="G"
```

Comment:
We are suggesting that it's the Sepolia test network here because of the color and it also matches the docs

```suggestion
      name="S"
```